src/metabase/util/fonts.clj

```diff
@@ -0,0 +1,41 @@
+(ns metabase.util.fonts
+  "font loading functionality."
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [metabase.util.files :as u.files]))
+
+(def font-path
+  "font resources path as a String."
+  "./resources/frontend_client/app/fonts/")
```

Comment:
It would probably be better to avoid hard-coding this as a relative path...usually when we access stuff in `resources/` we use `clojure.java.io/resource` to look up the resource on the classpath.

That returns a `java.net.URL` with the `file:` protocol though so I guess it doesn't play nice with directories...

I guess you could call `resource`, get the URL string with `.toString`, split it on `:`, take the second element and then pass that into `u.files/get-path`. That's still hacky though, maybe there's a better way.